README.md

```diff
@@ -1,51 +1,52 @@
-# OpenQL-Gym
-
-**_Note: This library is still under development and might change severely._**
-
-A Reinforcement Learning gym for training agents on problems that occur frequently in quantum compilation. This library
-is focussed around the specific compiler known as [OpenQL](https://github.com/QuTech-Delft/OpenQL).
-
-The gym supports two environments:
-1. Initial mapping: the problem of mapping virtual to physical qubits
-2. Scheduling: the problem of scheduling quantum gate operations such that hardware and commutation constraints are satisfied.
-
-## Tutorial notebooks
-We provide some tutorial notebooks to get to know Reinforcement Learning and this library. These notebooks are found in
-the folder [notebooks](notebooks).
-
-## Installation
-Below, we describe several steps for installing
-
-### Building and installing from source
-To build a wheel from this source one can run the command below.
-This will create a built wheel in a folder called `dist`.
-
-_Make sure that `pip`, `setuptools`, `wheel` are up-to-date._
-```commandline
-python setup.py bdist_wheel
+# QGYM – A Gym for Training and Benchmarking RL-Based Quantum Compilation
+`qgym` is a software framework that provides environments for training and benchmarking RL-based quantum compilers.
+It is built on top of OpenAI Gym and abstracts parts of the compilation process that are irrelevant to AI researchers.
+`qgym` includes three environments: Initial Mapping, Qubit Routing, and Scheduling, each of which is customizable and extensible.
```

Comment:
Maybe use the environment names as the packages are called:
InitialMapping, Routing, and Scheduling